manuscript/100_Analysis_In_TDD.md

```diff
@@ -42,7 +42,7 @@ Gherkin
 
 Hungry? Too bad, because the Gkerkin I�am gonna talk about is not edible. It is a�notation and a�way of thinking about behaviors of the specified piece of code. It can be applied on different levels of granularity -- any behavior, whether of a�whole system or a�single class, may be described using it.
 
-We actually talked about Gherkin before in this book, just did not name it. It is the GIVEN-WHEN-THEN structure that you can see everywhere, even in code samples as comments. This time, we are stamping a�name on it and analyzing it further.
+We talked about Gherkin before in this book, just did not name it. It is the GIVEN-WHEN-THEN structure that you can see everywhere, even in code samples as comments. This time, we are stamping a�name on it and analyzing it further.
```

Comment:
For me the text doesn't make it clear why you want to mention the name [Gherkin](https://github.com/cucumber/cucumber/wiki/Gherkin)  (here), it comes a bit as a surprise as Given-When-Then has been doing the job quite well thus far. 

Perhaps it deserves a short introduction that tells that Gherkin is a notation and at the same time the name of the language that Cucumber understands. And Cucumber is...

For now I just adapt the sentence:

"Although I didn't mention the Gherkin notation before, we did already use it. Gherkin is the GIVEN-WHEN-THEN structure that you can see everywhere, even as comments in the code samples."